docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -269,6 +293,65 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+Confluent Table Description Supplier
+------------------------------------
+The confluent table description supplier uses the `Confluent Schema Registry
+<https://docs.confluent.io/1.0/schema-registry/docs/intro.html>`_. to
+discover table definitions. The benefits of using the schema registry is that new tables can be defined without
+a cluster restart, schema updates will be detected and there is no need to define tables manually.
+
+========================================================== ======================================================================================================
+Property Name                                              Description
+========================================================== ======================================================================================================
```

Comment:
Lines here are too long, let's wrap the line somewhere. This go to entire change. Please make sure you wrap about 100.